test/integration/buildclient_test.go

```diff
@@ -195,13 +196,14 @@ func NewTestBuildOpenshift(t *testing.T) *testBuildOpenshift {
 
 	openshift.whPrefix = osPrefix + "/buildConfigHooks/"
 	osMux.Handle(openshift.whPrefix, http.StripPrefix(openshift.whPrefix,
-		webhook.NewController(clientWebhookInterface{osClient}, map[string]webhook.Plugin{
+		webhook.NewController(builder.OSClientBuildConfigClient{osClient}, builder.OSClientBuildClient{osClient}, map[string]webhook.Plugin{
```

Comment:
There's no good reason for "ClientWebhookInterface" to exist, all it did was provide CreateBuild methods, there's nothing unique about that that demands the webhook component have its own version of the interface.  Which is why i've removed it and replaced it with the more generic BuildClient interface to be used by things that want to create/manipulate Builds and BuildConfigClient interface for things that want to create/manipulate BuildConfigs